### ISSUE TITLE
Identifying CG or IG topic areas

### ISSUE THREAD

**Issue by egekorkan** [w3c/wot-marketing#330]
With the new WoT CG charter, we should discuss what topic areas should be moved to CG:

Opinions of people so far:
- @egekorkan : anything but website and plugfests can be CG tasks
- @Citrullin : IG Charter says that "Develop supporting materials such as implementation guidelines and tutorials" is IG activity

**Comment by Citrullin** [comment]
The scope of the IG charter also states "Reach out and collaborate with interested organizations, industry specific and infrastructure vendors, and communities in support of the Interest Group's mission".
This has to probably get clarified as well.

From my understanding the W3C CG will eventually not run under the new created non-profit. Once the non-profit is formed. The W3C can't provide the resources in order to maintain a chat system etc. That's why we should run these parts under the CG.
Maybe it makes sense to frame it somewhat in that direction. "Reach out and collaborate with interested organizations, industry specific and infrastructure vendors, and communities as representation of the W3C WoT WG."

While the CG has "reaches out and supports the development of a community not represented by the W3C WoT WG."

Something along that line. What do you think @egekorkan @relu91 ?

**Comment by egekorkan** [comment]
I think I agree but I am not sure understand every point properly. It would be difficult that a CG represents a WG given the political problems that it might cause.

**Comment by relu91** [comment]
I think this conflict with the IG that you pointed out also in the marketing call, might confuse people and we should somehow sort it out. 

> It would be difficult that a CG represents a WG given the political problems that it might cause.

What he is saying is that the IG should represent the WG while the CG will be a separate entity. 

I am ok with this differentiation as long as "the product" brand (i.e., Web of Things standards) will be consistent among the different groups.

**Comment by Citrullin** [comment]
> What he is saying is that the IG should represent the WG while the CG will be a separate entity.

Exactly. I agree with the part that it makes sense to be more involved in communities. And potentially having a matrix server etc.
We can't do that if we represent the WG in anyway. I understand the W3Cs position here.

> I am ok with this differentiation as long as "the product" brand (i.e., Web of Things standards) will be consistent among the different groups.

Good point. If we provide services etc. it has to be clear it's compatible with the WoT standards, but not a service by the W3C.
